pkg/utils/utils.go

```diff
@@ -205,3 +205,9 @@ func DownloadUrl(ctx context.Context, httpClient *http.Client, url string, local
 
 	return nil
 }
+
+func SanitizeString(str string) string {
```

Comment:
we can think of using replacer here https://pkg.go.dev/strings#NewReplacer
Could be common across function calls as seems to be safe for concurrent use by multiple goroutines.